shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/model/RouteDetailsStopListTest.kt

```diff
@@ -184,17 +184,27 @@ class RouteDetailsStopListTest {
         val mainStop = objects.stop { connectingStopIds = listOf(connectingStop.id) }
         val mainRoute = objects.route()
         val connectingRoute = objects.route()
-        objects.routePattern(connectingRoute) {
-            typicality = RoutePattern.Typicality.Typical
-            representativeTrip { stopIds = listOf(connectingStop.id) }
-        }
+        val pattern =
+            objects.routePattern(connectingRoute) {
+                typicality = RoutePattern.Typicality.Typical
+                representativeTrip { stopIds = listOf(connectingStop.id) }
+            }
 
         val globalData = GlobalResponse(objects)
 
         assertEquals(
             RouteDetailsStopList(
                 listOf(
-                    RouteDetailsStopList.Entry(mainStop, connectingRoutes = listOf(connectingRoute))
+                    RouteDetailsStopList.Segment(
+                        listOf(
+                            RouteDetailsStopList.Entry(
+                                mainStop,
+                                connectingRoutes = listOf(connectingRoute),
+                                patterns = listOf(),
```

Comment:
I would expect `patterns = listOf(pattern)` here; if it shouldn’t be there, there should probably be a comment explaining why.